### PR TITLE
Preload country list to speed up load times

### PR DESCRIPTION
**Pull request by jugarrit**
@chloepouprom @lemieux @mspensieri @dannytranlx 

The country dropdown would take too long to open when it was clicked. This will speed it up.

### CODE REVIEW

**Comment by lemieux** [comment]
Does it speed it up?

**Comment by jugarrit** [comment]
AHahah yes it seems so.
I'll try to make a gif comparison

**Comment by jugarrit** [comment]
This probably does nothing.

File: src/js/lib/react-telephone-input.jsx [hunk 1]
```diff
@@ -454,7 +457,7 @@ export class ReactTelephoneInput extends Component {
             }, function() {
                 self.scrollTo(self.getElement(self.state.highlightCountryIndex), true);
             });
-        }
```
Comment (dannytranlx) [inline src/js/lib/react-telephone-input.jsx]: This was needed.

**Comment by lemieux** [comment]
Yeah it probably does nothing

**Pull request closed without merging**